src/hb-ot-layout-gsub-table.hh

```diff
@@ -41,14 +41,22 @@ struct SingleSubstFormat1
   {
     TRACE_CLOSURE (this);
     Coverage::Iter iter;
+
+    hb_set_t closure_glyphs;
+    closure_glyphs.init ();
```

Comment:
I was actually thinking about doing this at the closure_context_t level (adding separate glyphs and out members) and update all closure implementations.  Can be done separately.  I don't mind doing it myself either.  Probably a swap() method as well that unions the out to the glyphs.  Humm, however, this will increase set operations we do significantly. I doubt it be a problem though.  Feel free to remove this commit so we merge the loop fix.

Reply:
Sounds good. I've removed it from this pr.